blueocean-github-pipeline/src/test/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubIssueTest.java

```diff
@@ -0,0 +1,123 @@
+package io.jenkins.blueocean.blueocean_github_pipeline;
+
+import com.cloudbees.hudson.plugins.folder.AbstractFolder;
+import com.google.common.base.Function;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import hudson.model.Job;
+import hudson.model.Run;
+import hudson.scm.ChangeLogSet;
+import hudson.util.DescribableList;
+import io.jenkins.blueocean.rest.factory.BlueIssueFactory;
+import io.jenkins.blueocean.rest.model.BlueIssue;
+import jenkins.branch.MultiBranchProject;
+import jenkins.plugins.git.GitSCMSource;
+import org.jenkinsci.plugins.github_branch_source.GitHubSCMSource;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.util.Collection;
+import java.util.Map;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class GithubIssueTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
```

Comment:
There is conflict of incompatible jetty libraries introduced in different versions of it and so wire mock and jenkins-test-harness do not play well by default. Basically we can't use default JenkinsRule. Replace this line with:

```
public JenkinsRule j = new PipelineBaseTest.PipelineBaseJenkinsRule();
```

Reply:
NICE! Thanks @vivek 💯